src/main/java/org/jboss/fuse/forge/addon/ui/FuseProjectCategory.java

```diff
@@ -16,16 +16,22 @@
 package org.jboss.fuse.forge.addon.ui;
 
 public enum FuseProjectCategory {
-    SPRING_BOOT("Spring Boot"),
-    KARAF("Karaf");
+    SPRING_BOOT("Spring Boot", "spring-boot"),
+    KARAF("Karaf", "jboss-fuse");
 
     private final String name;
+    private final String artifactIdPrefix;
```

Comment:
No, it's a prefix for an artifact id. I spotted some duplicate code so introduced a small refactoring to clean it up.